testing/postgresql-xl/APKBUILD

```diff
@@ -0,0 +1,248 @@
+# Maintainer: G.J.R. Timmer <gjr.timmer@gmail.com>
+pkgname=postgresql-xl
+_pkgname=${pkgname/ql/}
+pkgver=9.5r1.4
+pkgrel=0
+pkgdesc="Scalable Open Source PostgreSQL-based Database Cluster"
+url="http://www.postgres-xl.org/"
+arch="all"
+license="BSD"
+# Depends are deliberate, reason: avoid conflict with normal PostgreSQL
+depends="postgresql-xl-client postgresql-xl-libpq"
```

Comment:
What is the intention here? To pull packages from this aport, and it's to override the automatic soname dependencies?
It might be better to use `sonameprefix` and conflict with the regular postgresql package.